pkg/query-service/utils/format_test.go

```diff
@@ -449,7 +449,7 @@ func TestGetClickhouseColumnName(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			got := GetClickhouseColumnName(tt.typeName, tt.dataType, tt.field)
 			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("ClickHouseFormattedValue() = %v, want %v", got, tt.want)
+				t.Errorf("ClickHouseFormattedValue() = %v, want `%v`", got, tt.want)
```

Comment:
this is not required

Reply:
```suggestion
				t.Errorf("ClickHouseFormattedValue() = %v, want %v", got, tt.want)
```
> this is not required

thanks, I confirmed you're right @nityanandagohain

Reply:
fixed: [c7868d4](https://github.com/SigNoz/signoz/pull/6098/commits/c7868d425b47fd1801fe2e7ba5c6c64d86c0fddd)